src/Model/ImageFusion.py

```diff
@@ -366,7 +366,7 @@ def get_fused_window(level, window):
             get_fused_pixmap(old_images, fused_image[0], asp, i, "coronal",
                              windowing)
 
-    return color_axial, color_sagittal, color_coronal
+    return color_axial, color_sagittal, color_coronal, tfm
```

Comment:
The docstring for this function should be updated as you are returning a new variable.

Reply:
updated